cmd/explored/main.go

```diff
@@ -45,8 +45,9 @@ var cfg = config.Config{
 		EnableUPNP: false,
 	},
 	Scanner: config.Scanner{
-		Threads:             10,
+		BatchSize:           100,
```

Comment:
Going to leave this here but in `NewExplorer(cm ChainManager, store Store, batchSize int, scanCfg config.Scanner, log *zap.Logger) ` you could pass the indexerCfg instead of only the batch size to make it more consistent. This is partly why I suggest to rename `batchSize` because there's essentially two batchsizes and we can't immediately tell which one's used where.

Reply:
Resolved in followup